packages/lodestar-cli/src/cmds/beacon/cmds/run/index.ts

```diff
@@ -14,9 +14,18 @@ export const description = "Run a lodestar beacon node";
 
 export function builder(yargs: Argv<IBeaconArgs>): Argv<IBeaconArgs & Partial<IBeaconNodeOptions>> {
   const args = parseArgs(yargs);
-  return yargs
-    .options(beaconRunOptions)
-    .config(readBeaconConfig(args.config)) as unknown as Argv<IBeaconArgs & Partial<IBeaconNodeOptions>>;
+
+  let yargsConfig;
+  try {
+    yargsConfig = yargs
+      .options(beaconRunOptions)
+      .config(readBeaconConfig(args.config)) as unknown as Argv<IBeaconArgs & Partial<IBeaconNodeOptions>>;
+  } catch(error) {
+    // eslint-disable-next-line no-console
+    console.log(` ✖ ${error.message}\n`);
+  }
+
+  return yargsConfig;
```

Comment:
Is this type-safe? In case of error it will be returning an undefined object that the Typescript compilers assumes is a defined object.

Reply:
done